### DISCUSSION TITLE
Change the keys size in uiMidiKeyboard

### DISCUSSION THREAD

**Discussion by dervur** [RomanKubiak/ctrlr#665] [Q&A]
Hello there,

Is there a way to change the keys size in the keyboard? Making the modulator longer makes the keys longer, but making it larger shows only more keys. I would like to make the keys look larger if there is a way. I can't seem to find an option for it or maybe I'm blind, these are the only things I am able to change:
![image](https://github.com/RomanKubiak/ctrlr/assets/77355362/85f87552-ace3-4369-a2ca-d5992b9c283b)

Cheers

**Comment by Ted-juh** [comment]
If you mean the width of keys.. it's been set in Juce with setKeyWidth on the MidiKeyboardComponent, which to my knowledge should be available in Lua. But I believe you don't have the callback for those options in CTRLR, which means it "should" be added to the options pane in CtrlrX, if you want the easy way.

I can't test it right now if there's a workaround with Lua, not at home.

**Reply by dervur** [reply]
I tried in lua like this `panel:getModulatorByName("Keyboard"):setKeyWidth(10)` but it doesn't work (makes Ctrlr crash).
